spec/factories/bookings.rb

```diff
@@ -2,7 +2,7 @@
 
 FactoryGirl.define do
   factory :booking do
-    jms_booking_id "98765"
+    sequence(:jms_booking_id) { |n| "%5d" % n }
```

Comment:
http://batsov.com/articles/2013/06/27/the-elements-of-style-in-ruby-number-2-favor-sprintf-format-over-string-number-percent/